apps/weight-loss-consultant-users-mgnt-api/src/app/services/sorting-filtering.service.ts

```diff
@@ -75,14 +74,16 @@ export class SortingAndFilteringService {
         break;
       }
       default: {
-        const result = await this.trainerRepository.createQueryBuilder()
-          .where("gender = :gender", {gender: genderFilter === "Male" ? "Male" : "Female"})
+        const result = await this.trainerRepository.createQueryBuilder("trainer")
+          .leftJoinAndSelect("trainer.packages", "package")
+          .where("trainer.gender = :gender", {gender: genderFilter === "1" ? Gender.MALE : Gender.FEMALE})
           .orderBy(sortBy, order === "ASC" ? "ASC" : "DESC")
```

Comment:
What is "1" here means? Should separate to another variable or add comment.

Reply:
ok I will fix it